### PR TITLE
Ae fix up

### PR DESCRIPTION
**Pull request by paetling**
review @jsingle

### CODE REVIEW

File: tests/unit/commands/repos_test.py [hunk 1]
```diff
@@ -77,11 +77,11 @@ def test_override_repos_from_directory(self):
...
-        fake_update_local_repo.assert_called_once_with()
+        fake_update_local_repo.assert_has_calls([call(), call()])
 
     @patch('dusty.source.Repo.update_local_repo')
     def test_update_managed_repos_for_both(self, fake_update_local_repo):
         activate_bundle(['bundle-a'])
         activate_bundle(['bundle-b'])
         update_managed_repos()
-        fake_update_local_repo.assert_has_calls([call(), call()])
+        fake_update_local_repo.assert_has_calls([call(), call(), call()])
```
Comment (paetling) [inline tests/unit/commands/repos_test.py]: really weird that these did not start failing till i actually changed the values in here (these tests seem to never have been correct)

File: dusty/commands/repos.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def update_managed_repos():
     copy to bring it up to date with the latest master."""
     log_to_client('Pulling latest updates for all active managed repos:')
     overrides = set(get_config_value(constants.CONFIG_REPO_OVERRIDES_KEY))
-    for repo in get_all_repos(active_only=True):
+    specs_repo = get_specs_repo()
+    log_to_client('Updating managed copy of specs-repo before loading specs')
+    specs_repo.update_local_repo()
```
Comment (jsingle) [inline dusty/commands/repos.py]: we only want to do this if it's managed, right?

File: tests/unit/commands/repos_test.py (hunk 1 above)
Comment (jsingle) [inline tests/unit/commands/repos_test.py]: :space_invader:

File: dusty/commands/repos.py (hunk 2 above)
Comment (paetling) [inline dusty/commands/repos.py]: that is a good point.

**Pull request merged**